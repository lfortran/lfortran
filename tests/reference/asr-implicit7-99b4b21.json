{
    "basename": "asr-implicit7-99b4b21",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/implicit7.f90",
    "infile_hash": "6b139288be66d1f0911ec3534244dec1d912e697eec5c3efb28964a1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-implicit7-99b4b21.stderr",
    "stderr_hash": "7abfef1aadbe7259622aeca183a017c877b96ca8c42fb652e3640203",
    "returncode": 2
}