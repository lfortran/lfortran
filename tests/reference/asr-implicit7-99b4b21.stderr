semantic error: Implicit typing is not allowed, enable it by using --implicit-typing 
 --> tests/implicit7.f90:2:1
  |
2 | implicit real (n)
  | ^^^^^^^^^^^^^^^^^^ 
